spec/index.bs

```diff
@@ -1493,6 +1497,103 @@ interface GPUTextureUsage {
 };
 </script>
 
+### <dfn method for=GPUDevice>createTexture(descriptor)</dfn> ### {#GPUDevice-createTexture}
+
+<div algorithm="GPUDevice.createTexture">
+  **Arguments:**
+    - {{GPUTextureDescriptor}} |descriptor|
+
+  **Returns:** {{GPUTexture}}
+
+  1. If device is lost, or if this call doesn't follow the [$createTexture Valid Usage$], return an error texture.
+  1. Let |t| be a new {{GPUTexture}} object.
+  1. Set |t|.{{GPUTexture/[[textureSize]]}} to |descriptor|.{{GPUTextureDescriptor/size}}.
```

Comment:
It would be just easier to specify if you say the `descriptor` is copied by value into the internal slot.

Reply:
Unfortunately right now copying patterns from other parts of the spec doesn't work well because they haven't been copyedited.

I want to change more of the spec to use this pattern (just copying the whole descriptor directly into an internal slot).